webhook/cmd/start.go

```diff
@@ -43,13 +43,14 @@ func Start() {
 	}
 
 	options := webhook.Options{
-		ServiceName:    "events-webhook",
-		DeploymentName: "events-webhook",
-		Namespace:      namespace,
-		Port:           443,
-		SecretName:     "events-webhook-certs",
-		WebhookName:    "webhook.argo-events.argoproj.io",
-		ClientAuth:     tls.VerifyClientCertIfGiven,
+		ServiceName:     "events-webhook",
+		DeploymentName:  "events-webhook",
```

Comment:
`DeploymentName` also can be removed.

Reply:
It is still used [here](https://github.com/argoproj/argo-events/blob/master/webhook/webhook.go#L183)

Reply:
Silly me...